doc/tutorial/using_gpu.txt

```diff
@@ -39,6 +39,7 @@ file and run it.
 .. code-block:: python
 
     from theano import function, config, shared, sandbox
+    import theano.sandbox.cuda.basic_ops
```

Comment:
this one isn't needed. We don't call cuda.basic_ops there. Can you remove it?
thanks